etc/config.yaml.template

```diff
@@ -0,0 +1,4 @@
+# Peregrine Configuration
```

Comment:
I think this comment is unnecessary. If you think that it is unclear what the file is for, could you rename the file to `peregrine-config.yaml.template`?

Reply:
It's just normal to start YAML config files with a comment like that.